x-pack/plugins/alerting/public/services/maintenance_windows_api/get.ts

```diff
@@ -13,9 +13,11 @@ import { INTERNAL_BASE_ALERTING_API_PATH } from '../../../common';
 const rewriteBodyRes: RewriteRequestCase<MaintenanceWindow> = ({
   r_rule: rRule,
   category_ids: categoryIds,
+  scoped_query: scopedQuery,
   ...rest
 }) => ({
   ...rest,
+  scopedQuery,
```

Comment:
I am wondering if we should add `scopedQuery` when it can be undefined and just not have the value at all in the data model.

Reply:
Trying to fix this i opened a can of worms that is the frontend MW types, they need to be fixed so I will create another issue.